README.md

```diff
@@ -60,13 +60,6 @@ Default: `['@babel/plugin-proposal-optional-chaining', '@babel/plugin-proposal-n
 
 List of plugins to always include. Forwarded to [the corresponding option in `@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env#include).
 
-### `loose`
```

Comment:
also to chime in -- it might be a good idea to list the assumptions one would enable to try to recreate `loose` mode

Reply:
though it seems to affect so many that I'm not sure if it is a good idea now

Reply:
>Should we add some documentation about which assumptions we're explicitly not enabling by default?

@nabeel- I thought about this but they'll be adding several new assumptions in minor releases going forward so the list would probably end up drifting. I will add a comment that the current list is generally designed to match `loose` and I think future comments should mostly be added in terms of the allowlist.

>list the assumptions one would enable to try to recreate loose: false mode

@nikita-kensho To do this, you would disable all of the assumptions. I don't think this is useful to document explicitly because it's covered in the Babel docs and we don't currently use `loose: false` anywhere.

Reply:
> I will add a comment that the current list is generally designed to match loose and I think future comments should mostly be added in terms of the allowlist.

This makes sense to me. I just didn't want to remove this piece of documentation around `loose` when in practice the approximate same `loose: true` default behavior remains.